src/js/components/ServicesTable.js

```diff
@@ -294,6 +295,11 @@ var ServicesTable = React.createClass({
     let serviceStatus = service.getStatus();
     let serviceStatusClassSet = StatusMapping[serviceStatus] || '';
     let taskSummary = service.getTasksSummary();
+    let {TASK_RUNNING = 0} =
+        MesosSummaryStore.getFrameworkTasksByName(service.getName());
```

Comment:
Having this added to the count here seems to be the wrong approach.

Could we get ServiceList from CompositeState (which merges information from Marathon and Mesos) and get the count through there?